pyproject.toml

```diff
@@ -58,6 +58,7 @@ dev = [
     "pytest-examples>=0.0.14",
     "pytest-pretty>=1.2.0",
     "inline-snapshot>=0.23.0",
+    "dirty-equals>=0.9.0",
```

Comment:
This goes with the same line as `inline-snapshot`s. It's a utility package that helps a lot.

See: https://dirty-equals.helpmanual.io/latest/

Reply:
This is not necessary - but it did make the test cleaner. I can remove it if requested.